app.mjs

```diff
@@ -56,8 +56,10 @@ import {promises as fs} from 'fs';
             page.setUserAgent(userAgent);
 
             const screenshotFile = `output/${ticketNumber}/${contents.screenshot.filename}`;
-            console.log(`Saving screenshot to ${screenshotFile}`);
+            const screenshotFileFull = `output/${ticketNumber}/${contents.screenshot.filename.slice(0,-4)}Full.png`;
```

Comment:
Why are the last 4 characters being chopped off?

Reply:
I chopped of the last 4 (".png") because I wanted to add the "Full.png" suffix.

Reply:
Ah, that makes sense. Thanks!